auth/src/types/metadata.ts

```diff
@@ -0,0 +1,47 @@
+import { CountryCode, NSACodes } from "./countrycodes";
+
+/**
+ * User specific data
+ */
+export interface User {
+  id: string;
+  email: string;
+  email_verified: boolean;
+  family_name: string;
+  given_name: string;
+  locale: string;
+  pronouns: string;
+}
+
+export type Conference = number;
+export type Commitee = {
+  id: string;
+  conference: Conference;
+};
```

Comment:
Doesn't the committee need a name?

Reply:
It does. In the future it would probably make sense to include the name but also the other data about the conference and its structure in an own application just for this purpose. Currently we probably only need the id to check against.